saleor/graphql/channel/mutations/channel_create.py

```diff
@@ -221,6 +229,13 @@ def clean_input(cls, info: ResolveInfo, instance, data, **kwargs):
                     "default_transaction_flow_strategy"
                 ] = default_transaction_strategy
 
+            if allow_to_create_order_without_payment := order_settings.get(
```

Comment:
What if `allow_to_create_order_without_payment` will be `False`?
It looks like we are not able to set up the `False` value. It means that we're missing unit tests.